src/frontend/api.ts

```diff
@@ -346,6 +346,7 @@ function createMapServerApi() {
     // Return the url to a map style from the map server
     getStyleUrl: (id: string): string | undefined =>
       mapServerPort ? `${getBaseUrl(mapServerPort)}styles/${id}` : undefined,
+    getTileset: async (url: string) => await (await ky.get(url)).json(),
```

Comment:
Is there a better way to do this?

Reply:
yeah i'd update to this for now for the sake of consistency with how the other endpoints are set up:

```suggestion
    getTileset: async (id: string): Promise<TileJSON> =>
      (await guaranteeClient().get(`tilesets/${id}`)) as TileJSON,
```

i think we can make improvements to the overall setup of the API for this but not within the scope of this PR.

Reply:
but the problem is that I can get the whole url from the `sources`. I suppose I could try to parse just the ID...but that seems to be a little circular given that there actually is no straightforward way to get the tileset Id

Reply:
yeah i see what you mean (i'm responding to your other question acknowledging this). i think i'm okay with the circularness for the sake of API consistency :)